### PR TITLE
DDL for block_detail.sql WJT 31/01/2018

### PR DESCRIPTION
**Pull request by williamthurley**
Provisional request, clearly have at least one edit to make, more for checking for other points.

### CODE REVIEW

File: src/main/sql/block_detail.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
+CREATE TABLE block_detail                                                          
+(                                                                              
+--Primary Key                                                                                         
+  PRIMARY KEY (block_id,entry_number),                                             
+                                                                         
+--Columns                                                              
+  block_id INT NOT NULL,
+  entry_number INT NOT NULL,
+  cash_value NUMERIC(10,2) NOT NULL DEFAULT 0,
+  user_id (whatever format user_detail has),
```
Comment (williamthurley) [inline src/main/sql/block_detail.sql]: Need to collab with Sean to fill this bit in properly.

**Review by williamthurley — COMMENTED** [review]
user_id change required. Please see comments.

File: src/main/sql/block_detail.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  block_detail_description VARCHAR(200) NOT NULL DEFAULT ' ',
+  written_off CHAR(1) NOT NULL DEFAULT 'Y',
+  currency_code CHAR(3) NOT NULL,
+  ledger_id CHAR(10) NOT NULL,
+
+--Constraints
+  CONSTRAINT written_off_is_boolean
```
Comment (gabewilliam) [inline src/main/sql/block_detail.sql]: Maybe call this constraint something else? Technically the field isn't a boolean type

**Review by gabewilliam — APPROVED** [review]
All good JT

**Pull request merged**